rust/scx_utils/src/topology.rs

```diff
@@ -126,6 +126,7 @@ pub struct Cpu {
     pub core_id: usize,
     pub llc_id: usize,
     pub node_id: usize,
+    pub hw_id: usize,
```

Comment:
Why is `cpu.hw_id` necessary?

Reply:
In layered it uses the Cpu struct to lookup the core cpumask